source/camera/basic/capture_from_file.py

```diff
@@ -10,7 +10,13 @@ def _main():
     app = zivid.Application()
     camera = app.create_file_camera(Path() / get_sample_data_path() / "FileCameraZividOne.zfc")
 
-    with camera.capture() as frame:
+    settings = zivid.Settings(
+        acquisitions=[
+            zivid.Settings.Acquisition(aperture=5.66, exposure_time=datetime.timedelta(microseconds=8333),),
+        ],
+    )
```

Comment:
Also change here if you decide to change.

Reply:
Done